<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Btn_Reset">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>초기화</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_OK">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>확인</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Cancel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>취소</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>351</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox_1">
      <item>
       <property name="text">
        <string>Home Key</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>End Key</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>page up Key</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>page down Key</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>오른팔 중간</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>오른팔 위</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>왼팔 위</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>왼팔 중간</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>왼팔 아래</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>오른팔 아래</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_2">
      <item>
       <property name="text">
        <string>Home Key</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>End Key</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>page up Key</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>page down Key</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_3">
      <item>
       <property name="text">
        <string>Home Key</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>End Key</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>page up Key</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>page down Key</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBox_4">
      <item>
       <property name="text">
        <string>Home Key</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>End Key</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>page up Key</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>page down Key</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="comboBox_5">
      <item>
       <property name="text">
        <string>Home Key</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>End Key</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>page up Key</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>page down Key</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QComboBox" name="comboBox_6">
      <item>
       <property name="text">
        <string>Home Key</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>End Key</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>page up Key</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>page down Key</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
